helm/charts/kratos/templates/deployment-kratos.yaml

```
@@ -192,7 +192,7 @@ spec:
             {{- toYaml .Values.deployment.customLivenessProbe | nindent 12 }}
           {{- else }}
             httpGet:
-              path: /admin/health/ready
+              path: /health/alive
```

The default path for the admin api is `/admin` calling `/health/alive` results in a `307 Temporary Redirect` to `/admin/health/alive`.

@piotrmsc This pr solely switches to the correct alive probe endpoint [as documented](https://github.com/ory/kratos/blob/b75313e15d7071a77ac49a8425c1fac1be1f5bb3/.schema/openapi.json#L1366-L1444). There is no security aspect to it. 

```suggestion
              path: /admin/health/alive
```